receiver/vcenterreceiver/metadata.yaml

```diff
@@ -296,6 +304,50 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.resource_pool.memory.swapped:
+    enabled: false
+    description: The portion of memory, in MB, that is granted to a virtual machine from the host's swap space.
+    unit: MiBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: []
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.105.0"
+  vcenter.resource_pool.memory.ballooned:
+    enabled: false
+    description: The size of the balloon driver in a virtual machine, in MB. The host will inflate the balloon driver to reclaim physical memory from a virtual machine.
```

Comment:
Most if not all of these descriptions might be a little wordy. VMs already have examples here for `swapped` and `ballooned`, so I'd take a look at those. If you want to add some extra detail I think that is still OK, but maybe it makes more sense to do it under the `extended_documentation`?